src/devices/video/mc6847.cpp

```diff
@@ -406,6 +432,9 @@ inline void mc6847_friend_device::next_scanline()
 		m_logical_scanline_zone = SCANLINE_ZONE_BOTTOM_BORDER;
 		enter_bottom_border();
 	}
+
+	// This helps to confirm synchronization between the screen device and the 6847
+	LOGMASKED(LOG_NEXTLINE, "mc6847_friend_device::next_scanline(): (vpos,hpos)=(%d,%d), m_physical_scanline='%d', m_logical_scanline_zone='%s', m_logical_scanline='%d'\n", (screen().vpos()), (screen().hpos()), m_physical_scanline, (scanline_zone_string((scanline_zone)m_logical_scanline_zone)), m_logical_scanline);
```

Comment:
Besides the comment being pointless, this line is ridiculously long and has redundant parentheses.

Reply:
yup.  Comment is gone, and the LOGMASKED call has been broken into multiple lines.